src/NUnitEngine/nunit.engine.tests/Services/DefaultTestRunnerFactoryTests.cs

```diff
@@ -89,6 +89,9 @@ public void ServiceIsStarted()
         [TestCase("x.dll y.dll z.dll",     "Multiple", typeof(MultipleTestProcessRunner))]
         public void CorrectRunnerIsUsed(string files, string processModel, Type expectedType)
         {
+            if (files == "EngineTests.nunit")
+                files = Path.Combine(AssemblyDirectory, files);
```

Comment:
Rather than writing code to figure out the directory, use TestContext.CurrentContext.TestDirectory